packages/app/src/cli/services/dev.ts

```diff
@@ -265,14 +265,16 @@ async function dev(options: DevOptions) {
       envIdentifiers: prodEnvIdentifiers,
     })
 
+    await updateAppIdentifiers({app: localApp, identifiers, command: 'deploy'})
```

Comment:
this is because this function only writes to the `.env` file if the command is `deploy`. This is something to improve after editions.